common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -553,4 +554,13 @@ public String getCrowdNodePrimaryAddress() {
     public void setCrowdNodePrimaryAddress(@NonNull String address) {
         prefs.edit().putString(PREFS_KEY_CROWDNODE_PRIMARY_ADDRESS, address).apply();
     }
+
+    @NonNull
+    public Float getPrefsKeyCrowdNodeStakingApy() {
+        return prefs.getFloat(PREFS_KEY_CROWDNODE_STAKING_APY, 0.0f);
+    }
+
+    public void setPrefsKeyCrowdNodeStakingApy(@NonNull float apy) {
+        prefs.edit().putFloat(PREFS_KEY_CROWDNODE_STAKING_APY, apy).apply();
+    }
```

Comment:
To save the last obtained value of the APY.  This way a value can be displayed in the event of a blockchain rescan, wallet reset or network offline.